tensorflow/lite/micro/kernels/depth_to_space_test.cc

```diff
@@ -0,0 +1,110 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
```

Comment:
we're hitting an annoying discrepancy between internal and extenal clang-formatting.

The solution here is to delete the gmock and gtest includes.

The underlying problem is that when the code is imported internally, these includes become:

```cc
#include "/internal/path/to/gmock/gmock.h"
```

and then clang-format wants them to be in the lower block (i.e. next to `"flatbuffers/flatbuffers.h"`.

So the internal and external ordering is conflicting.

We only recently added clang-format checks externally which might explain why we haven't seen this before.

Reply:
Okay, attempting in c30b352.

Reply:
https://github.com/tensorflow/tensorflow/pull/46298 will help catch this via the external CI.